west.yml

```diff
@@ -111,7 +111,7 @@ manifest:
     - name: hostap
       repo-path: sdk-hostap
       path: modules/lib/hostap
-      revision: d82c309c2e7eea5f08bbc1ac9e2f4c4622fbe998
+      revision: vX.Y.Z-ncsN-snapshot1
```

Comment:
Need version details for hostap

Reply:
@sachinthegreen  ?

Reply:
as discussed, `2.11-devel` is the unreleased version in hostapd.

Reply:
then how do we mark it @carlescufi  ?

Reply:
its already marked as `v2.11-devel-ncs1-snapshot1` which is fine.